web-jehlomat/src/screens/SeznamUzivatelu/SeznamUzivatelu.tsx

```diff
@@ -1,14 +1,16 @@
 import { FC, useState } from 'react';
 import styled from '@emotion/styled';
 import SearchInput from '../../Components/Inputs/SearchInput/SearchInput';
-import AddButton from '../../Components/Buttons/AddButton/AddButton';
 import { usersMock } from './usersMock';
 import { grey } from '../../utils/colors';
 import ListItem from '../../Components/List/ListItem/ListItem';
 import { useHistory } from 'react-router';
 import { Header } from '../../Components/Header/Header';
-import StyledModal from '../../Components/Modal/StyledModal';
-import PridatUzivateleModal from '../../Components/Modal/PridatUzivateleModal';
+import { useMediaQuery } from '@mui/material';
+import AddButton from '../../Components/Buttons/AddButton/AddButton';
+import TextButton from '../../Components/Buttons/TextButton/TextButton';
+import { media } from '../../utils/media';
+import { primaryDark } from '../../utils/colors';
 
 interface Props {}
```

Comment:
Empty props, similar to the `PridatUzivateleForm.tsx`

Reply:
Sice je better practice nedefinovat, eslint pravidlo na toto nemame, osobne za mne ok

Reply:
Neresme, jinak se projekt nikdy nedodela